core/app/models/mno_enterprise/subscription_event.rb

```diff
@@ -0,0 +1,20 @@
+module MnoEnterprise
+  class SubscriptionEvent < BaseResource
+    property :created_at, type: :time
+    property :updated_at, type: :time
+
+    property :uid, type: :string
+    property :status, type: :string
+    property :event_type, type: :string
+    property :message, type: :string
+    property :provisioning_data
+
+    has_one :subscription
+
+    def to_audit_event
```

Comment:
Can you add `organization_id` to the `Hash`?
It's used to link Audit Event to their Org in the Admin Panel